src/editorview.cpp

```diff
@@ -177,6 +187,28 @@ void EditorView::drawForeground(QPainter* p, const QRectF& rect)
   */
 }
 
+void EditorView::contextMenu(QMouseEvent* event)
```

Comment:
It would be more traditional to have this be `const QPoint& pos` so you can open the menu with something besides a mouse event, but this is just temporary anyway so I'm not worried about it. (Unless there's going to be a context menu with other stuff in it later, but we can deal with cleaning it up then.)